File: Data Structures & Algorithms/sort-an-array/submission-0.java
class Solution {
    public int[] sortArray(int[] nums) {
        int n=nums.length;
        for(int i=0;i<n;i++){
            int j=i;
            while(j>0 && nums[j]<nums[j-1]){
                int temp=nums[j-1];
                nums[j-1]=nums[j];
                nums[j]=temp;
            j-=1;
            }
        }
            return nums;
        }
    }

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        d={}
        c=0
        n=len(nums)
        for i in nums:
            if i not in d:
                d[i]=1
                nums[c]=i
                c+=1
            else:
                d[i]+=1
        return c
